sixtyfps_runtime/rendering_backends/gl/lib.rs

```diff
@@ -677,120 +608,124 @@ impl ItemRenderer for GLItemRenderer {
             .unwrap()
             .as_font()
             .clone();
-        let font_height = font.height();
 
         let paint = match self.brush_to_paint(
             text_input.color(),
             &mut rect_to_path(item_rect(text_input, self.scale_factor)),
         ) {
-            Some(paint) => paint,
+            Some(paint) => font.init_paint(text_input.letter_spacing() * self.scale_factor, paint),
             None => return,
         };
 
-        let letter_spacing = text_input.letter_spacing() * self.scale_factor;
-        let horizontal_alignment = text_input.horizontal_alignment();
-        let vertical_alignment = text_input.vertical_alignment();
-        let metrics = self.draw_text_impl(
-            width,
-            height,
-            sixtyfps_corelib::items::TextInput::FIELD_OFFSETS.text.apply_pin(text_input),
+        let (min_select, max_select) = text_input.selection_anchor_and_cursor();
+        let cursor_pos = text_input.cursor_position();
+        let cursor_visible = cursor_pos >= 0 && text_input.cursor_visible();
+        let mut canvas = self.shared_data.canvas.borrow_mut();
+        let font_height = canvas.measure_font(paint).unwrap().height();
```

Comment:
Just a suggestion, but since the same metric is also used in `layout_text_lines` perhaps it should be called `line_height`? 

The variable could also probably be declared and initialised inside the closure from the `metrics` parameter, unless I'm missing something?

Reply:
You're right, it is not the same.